etc/anomaly_detection.conf

```diff
@@ -5,14 +5,19 @@ max_logfile_count = 10
 max_logfile_size_mb = 200
 log_rotation_type = "interval"
 logging_default_format_string = "%(asctime)s - %(levelname)s - %(filename)s[:%(lineno)d] - %(message)s"
+# debug = true
 
 [database]
 connection=sqlite:///anomaly_detection.db
 backend = "sqlalchemy"
 
 [training]
-dataset_source_type=database
+dataset_source_type=csv
```

Comment:
I think the database type should be kept but commented out with comments on how to use database for training.